### ISSUE TITLE
Accessibility - Information Screen PDFs Not Being Read by TalkBack

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#66]
Because of a technology issue between accessible PDFs and the operating system TalkBback cannot read the PDFs content correctly. By that I mean if you set up headings or data table or have images TalkBack will not read them as such but only as content with no semantic meaning. Screen reader users can use headings to navigate around a page as a sighted user would look for clues visually to find certain things.

I'm going to close the older issue #21, so we don't have all these extra notes to wade through. 

We will keep this issue open to check back now and again to see if Android has corrected the problem.

@carolynhuang,

Can you send me the PDFs in the More Information section so I can verify that they are accessible on my desktop?

I have not looked at any of them in a year or more, and some are new. I want to make sure they are Section 508 compliant for when the Android and TalkBack problem is corrected.

**Comment by jfc3-dol** [comment]
When I attempt to read some of the PDFs, it does not open the file and gives me a message telling me the following: "Can not display (PDF NAME GOES HERE) cannot be opened."

But some of the older PDFs don't get this message but allow me to open them in an external application. A PDF that will open up is the "TDA FQAs" or the "EO FQAs". The opens that don't even open are "Child Labor Report" or "Child and Forced Labor List of Goods" or the "Forced Child Labor List of Products".

**Comment by jfc3-dol** [comment]
We used our Samsung Galaxy S7 (SM-G930V) to do this testing with.

**Comment by jfc3-dol** [comment]
Waiting for the final updated PDFs to be added to the application.

**Comment by jfc3-dol** [comment]
Closing issue because Android and iOS still don't read the structure (headings, TABLE, etc.) from PDFs.
